vultr/downloader.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# https://docs.vultr.com/vultr-ip-space
+# https://geofeed.constant.com/?json
+
+set -euo pipefail
+set -x
+
+
+# get from public ranges
+curl -s https://geofeed.constant.com/?text > /tmp/vultr.txt
+
+#save ipv4
+REGEX_V4="^(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9][0-9]?|0)\.(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9][0-9]?|0)\.(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9][0-9]?|0)\.(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9][0-9]?|0)(\/([1-9]|[1-2][0-9]|3[0-2])){0,1}$"
```

Comment:
too complicated and unreliable

example: https://github.com/lord-alfred/ipranges/blob/main/digitalocean/downloader.sh#L13-L17

Reply:
Just a "simple" IPv4 / v6 Filtering Regex. But okay, then I'll do like digitalocean.